src/middlewared/middlewared/etc_files/ctdb_private.conf.mako

```diff
@@ -0,0 +1,15 @@
+<%
+    from middlewared.plugins.smb import SMBHAMODE
+
+    clustered = SMBHAMODE[middleware.call_sync('smb.get_smb_ha_mode')] == SMBHAMODE.CLUSTERED
+    if not clustered:
+        return
+
+    data = middleware.call_sync('datastore.query', 'ctdb.private.ips')
```

Comment:
Why not `ctdb.private.ips.query` ?

Seems more future proof than directly querying database